buildconfig/stubs/pygame/scrap.pyi

```diff
@@ -1,10 +1,11 @@
-from typing import AnyStr, List
+from typing import List, Optional
+from collections.abc import ByteString
 
 def init() -> None: ...
 def get_init() -> bool: ...
-def get(data_type: str) -> AnyStr: ...
+def get(data_type: str) -> Optional[bytes]: ...
 def get_types() -> List[str]: ...
-def put(data_type: str, data: AnyStr) -> None: ...
+def put(data_type: str, data: ByteString) -> None: ...
```

Comment:
Why is this ByteString and not just bytes? What is ByteString?

Reply:
`ByteString` is a generic bytes-like object (which in addition to `bytes`, also accepts few more things like `bytearray` and the like)

I'm using this because I observed the C code for `put` use `y#` format which should handle the same

Reply:
Linking you to a discussion I found: https://github.com/python/cpython/issues/91896
I'll support proceeding the current way or another way if that changes your mind.

Reply:
sounds like we are still waiting for Buffer to make it from PEP into CPython. I guess we should roll with this as it is for now and revisit when Buffer turns up in our lowest version in a few years?